includes/class-cache.php

```
@@ -176,6 +184,67 @@ public function __construct( array $options = array() ) {
+			$user = wp_get_current_user();
+			if ( empty( $user->roles ) ) {
+				return false;
+			}
+
+			foreach ( $user->roles as $role ) {
+				if ( in_array( $role, $allowed_roles, true ) ) {
+					return true;
+				}
+			}
+			return false;
+		}
```

PHPMD (2.15.0)

[error] The method is_cache_allowed_for_current_user is not named in camelCase. (undefined)

(CamelCaseMethodName)